standalone/src/models/product-type/attribute-definition/fields-config.ts

```diff
@@ -39,6 +39,7 @@ export const restFieldsConfig: TModelFieldsConfig<TAttributeDefinitionRest> = {
     ),
     label: fake(() => LocalizedString.random()),
     inputTip: fake(() => LocalizedString.random()),
+    level: oneOf(...Object.values(levels)),
```

Comment:
Based on the conversation I had with @jaikumar-tj when I migrated this model, the `level` field should have the `levels.Variant` value by default.